test/residency/contract/test_generated_contract.py

```
@@ -0,0 +1,1036 @@
+        "authority_transaction_result"
+    ]["conditionals"][0]
+    predicate_conditional["if"] = {"contexts": ["health_read"]}
+    try:
+        registry_module.validate_contract_registry(predicate_registry)
+    except ValueError as error:
+        require(
+            "predicate is not translatable" in str(error),
+            "schema predicate mutation diagnostic drifted",
+        )
+    else:
+        raise AssertionError("non-translatable schema predicate was accepted")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)